galeracluster/source/documentation/install-mysql.rst

```diff
@@ -224,25 +228,13 @@ For Galera Cluster 8.4:
 
    apt-get install galera-4 galera-arbitrator-4 mysql-wsrep-8.4
 
-If you want to install version 4 of Galera Cluster, you would instead execute the following from the command-line:
-
-.. code-block:: console
-
-   apt-get install galera-4 galera-arbitrator-4 mysql-wsrep-8.0
-
 On servers using the ``yum`` package manager (i.e., Red Hat Enterprise Linux and CentOS distributions), you would instead execute this command:
 
 .. code-block:: console
 
-   yum install galera-3 mysql-wsrep-8.4
-
-Please note that on CentOS 6 and 7, this command may generate a transaction check error. For more information on that error and how to resolve it, see the section below on :ref:`MySQL Shared Compatibility Libraries <centos-mysql-shared-compt>`.
-
-If you want to install version 4 of Galera Cluster using ``yum``, you would instead execute the following from the command-line:
-
-.. code-block:: console
+   yum install galera-4 mysql-wsrep-8.4
```

Comment:
please do not remove 8.. for 8.0 the 8.0 insturctions must exist. for 8.4 this is ok

Reply:
Fixed